pyasic/config/pools.py

```diff
@@ -107,6 +107,10 @@ def from_dict(cls, dict_conf: Union[dict, None]) -> "Pool":
     @classmethod
     def from_api(cls, api_pool: dict) -> "Pool":
         return cls(url=api_pool["URL"], user=api_pool["User"], password="x")
+    
+    @classmethod
+    def from_epic(cls, api_pool: dict) -> "Pool":
+        return cls(url=api_pool["pool"].replace("stratum+tcp://", ""), user=api_pool["login"], password=api_pool["password"])
```

Comment:
Don't replace the prefix here, want to keep everything for the URL.

Might be possible at a later point to split the URL into its components, but not really worth doing yet.